tests/test_vector2_dot_product.py

```diff
@@ -0,0 +1,11 @@
+import ppb_vector
+
```

Comment:
This should be two lines of white space. Please refer to [PEP8](https://www.python.org/dev/peps/pep-0008/) for style guide.